### PR TITLE
VeriBE followup

### PR DESCRIPTION
**Pull request by marioba**
VeriBE followup:
- import / export 'maengel' shapefile
- fixed missing symbols
- fixed some little errors in checks
- fixed wms links
- new check 'PNF'
- changes in 'maengel'
- added tolerance in 'isolierte übrige Gebäudeteile' check

### CODE REVIEW

File: modules/veribe_ee/checks/gebaeudeadressen/gebaeudeadressen_lokalisation.py [hunk 1]
```diff
@@ -238,20 +239,23 @@ def run(self):
 
             # only one feature is selected
             for feature in iter:
+                print('in for')
```
Comment (mbernasocchi) [inline modules/veribe_ee/checks/gebaeudeadressen/gebaeudeadressen_lokalisation.py]: is this debugging code? you can use the qgis log if you want to show debugging messages to the user

File: modules/veribe_ee/checks/gebaeudeadressen/gebaeudeadressen_lokalisation.py [hunk 2]
```diff
@@ -238,20 +239,23 @@ def run(self):
...
                 prinzip_idx = vlayer_lokalisation.fieldNameIndex("nummerierungsprinzip_txt")
                 attributeprovisorisch_idx = vlayer_lokalisation.fieldNameIndex("attributeprovisorisch_txt")
                 offiziell_idx = vlayer_lokalisation.fieldNameIndex("istoffiziellebezeichnung_txt")
                 status_idx = vlayer_lokalisation.fieldNameIndex("status_txt")
                 inaenderung_idx = vlayer_lokalisation.fieldNameIndex("inaenderung_txt")
                 art_idx = vlayer_lokalisation.fieldNameIndex("art_txt")
-                
+
+                print('prinzip_idx', prinzip_idx)
```
Comment (mbernasocchi) [inline modules/veribe_ee/checks/gebaeudeadressen/gebaeudeadressen_lokalisation.py]: debugging code?

File: modules/veribe_ee/checks/gebaeudeadressen/gebaeudeadressen_lokalisation.py [hunk 3]
```diff
@@ -238,20 +239,23 @@ def run(self):
...
                 if prinzip_idx == -1 or attributeprovisorisch_idx == -1 or offiziell_idx == -1 or status_idx == -1 or inaenderung_idx == -1 or art_idx == -1:
                     self.iface.messageBar().pushMessage("Error",  _translate("VeriSO_EE_Geb_LokTest", "Field not found.", None), level=QgsMessageBar.CRITICAL, duration=10)                                                                        
                     QApplication.restoreOverrideCursor()
                     return
 
                 prinzip = feature.attributes()[prinzip_idx]
-                attributeprovisorisch = feature.attributes()[attributeprovisorisch_idx]            
+                print('prinzip', prinzip)
```
Comment (mbernasocchi) [inline modules/veribe_ee/checks/gebaeudeadressen/gebaeudeadressen_lokalisation.py]: debugging code?

File: modules/veribe_ee/checks/pnf/pnf.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+            layer["readonly"] = True
+            layer["featuretype"] = "bodenbedeckung_boflaeche"
+            layer["geom"] = "geometrie"
+            layer["key"] = "ogc_fid"
+            layer["group"] = group1
+            layer["sql"] = "art in (14, 15, 16)"
+            layer["style"] = "pnf/gewaesser.qml"
+            vlayer = self.layer_loader.load(layer)
+
+            layer = {}
```
Comment (mbernasocchi) [inline modules/veribe_ee/checks/pnf/pnf.py]: you could also do (no need to do it here, just saying)
```
layer = {"type": "postgres",
             "title": mytitle}
```

File: modules/tools/exportdefectsshp.py [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+                                          "postgres")
+
+            uri = QgsDataSourceURI()
+            uri.setConnection(db_host, db_port, db_name, db_user, db_pwd)
+            uri.setDataSource(db_schema, "t_maengel_polygon", "the_geom", "",
+                              "ogc_fid")
+            vlayer_polygons = QgsVectorLayer(uri.uri(), "Maengel (Linien)",
+                                             "postgres")
+
+            if not vlayer_points.isValid():
```
Comment (mbernasocchi) [inline modules/tools/exportdefectsshp.py]: 3 time the same code is a good candidate for a small function. maybe even to be put in utils

File: modules/tools/importdefects.py [hunk 6]
```diff
@@ -63,14 +64,93 @@ def init_gui(self):
...
+
+        QApplication.restoreOverrideCursor()
+        self.close()
+
+
+    def import_shp(self, shp, load_checked):
+        shp =  self.lineEditDefectsFile.text().strip()
+        load_checked = self.load_defects_check.isChecked()
+
+        sf = shapefile.Reader(shp)
```
Comment (mbernasocchi) [inline modules/tools/importdefects.py]: why don't you just read the file using qgis api?
```layer = QgsVectorLayer(shp, "importlayer_shp", "ogr")

Comment (marioba) [inline modules/tools/importdefects.py]: @mbernasocchi refactored import_shp(). A little more clean and pythonic now..

**Pull request merged**